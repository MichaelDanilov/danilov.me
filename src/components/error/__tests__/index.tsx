import React from 'react';
import renderer from 'react-test-renderer';
import Error from '..';

import Languages from '../../../constants/languages';

describe('Error', () => {
  it('english version renders correctly', () => {
    const tree = renderer.create(<Error lang={Languages.english} />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('russian version renders correctly', () => {
    const tree = renderer.create(<Error lang={Languages.russian} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
