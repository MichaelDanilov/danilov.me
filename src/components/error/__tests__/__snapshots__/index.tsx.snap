// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error english version renders correctly 1`] = `
<div
  className="sc-bdfBwQ gSQAtd"
>
  <p
    className="sc-gsTCUz bGwQZY"
    dangerouslySetInnerHTML={
      Object {
        "__html": "Looks like, something has gone wrong",
      }
    }
  />
</div>
`;

exports[`Error russian version renders correctly 1`] = `
<div
  className="sc-bdfBwQ gSQAtd"
>
  <p
    className="sc-gsTCUz bGwQZY"
    dangerouslySetInnerHTML={
      Object {
        "__html": "Кажется что-то пошло не так",
      }
    }
  />
</div>
`;
